docs/visualize/aggregations.asciidoc

```diff
@@ -1,36 +1,51 @@
 [[supported-aggregations]]
 == Supported aggregations
 
-Use the supported aggregations to build your visualizations.
+To build your visualizations, use the supported aggregations. Each visualization supports a subset of aggregations. 
 
 [float]
 [[visualize-metric-aggregations]]
 === Metric aggregations
 
 Metric aggregations extract field from documents to generate data values.
```

Comment:
extract fields or extract a field?

The tables or so much easier to read!  Big improvement!